src/main/webapp/app/entities/drug/drug-update.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="row justify-content-center">
```

Comment:
Should be deleted

Reply:
done

Reply:
also the warning showing on the test file were fixed